### DISCUSSION TITLE
Event Camera Python API error

### DISCUSSION THREAD

**Discussion by Hiromaro-Tanii** [microsoft/AirSim#4827] [Support Q&A]
I want to use Event Camera, but I get an error like. Why?

AirSim:Latest
Unreal Engine:4.27
Visual Studio:2022
Python:3.7/3.9

Initialized event camera simulator with sensor size: (64, 64)

DeprecationWarning: The binary mode of fromstring is deprecated, as it behaves surprisingly on unicode inputs. Use frombuffer instead
np.fromstring(response[0].image_data_uint8, dtype=np.uint8), event_generator.rgb_image_shape,)

D:\ANACONDA\envs\ASclient\lib\site-packages\numba\core\typed_passes.py:329: NumbaPerformanceWarning:
The keyword argument 'parallel=True' was specified but no transformation for parallel execution was possible.

To find out why, try turning on parallel diagnostics, see https://numba.readthedocs.io/en/stable/user/parallel.html#diagnostics for help.

File "event_simulator.py", line 24:
@njit(parallel=True)
def esim(
^

warnings.warn(errors.NumbaPerformanceWarning(msg,

\PythonClient\eventcamera_sim\test_event_sim.py:91: DeprecationWarning: The binary mode of fromstring is deprecated, as it behaves surprisingly on unicode inputs. Use frombuffer instead
np.fromstring(response[0].image_data_uint8, dtype=np.uint8), event_generator.rgb_image_shape,)

**Comment by Adurisitooo13** [comment]
Did you get an answer? I'm having the same issue.
